client/src/index.js

```diff
@@ -1,8 +1,163 @@
-import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import React from 'react';
+import './Login.css';
+
+//from material ui
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Link from '@material-ui/core/Link';
+
+const styles = makeStyles(theme => ({
+    root: {
+        height: '100vh',
+    },
+    paper: {
+        margin: theme.spacing(25, 4),
+        isplay: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    },
+    image: {
+        backgroundImage: 'url(https://images.unsplash.com/photo-1493770348161-369560ae357d?ixlib=rb-1.2.1&auto=format&fit=crop&w=1000&q=80)',
+        backgroundRepeat: 'no-repeat',
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+    },
+    button: {
+        margin: theme.spacing(3, 0, 2),
+    },
+    form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing(1),
+    },
+  }));
+
+export default function Login() {
+
+    const state = {
+        email: "",
+        password: "",
+        confirmPassword: ""
+    };
+
+    const updateEmail = e => {
+        state.email = e.target.value
+    }
+    
+    const updatePassword = e => {
+        state.password = e.target.value
+    }
+
+    const updateConfirmPassword = e => {
+        state.confirmPassword = e.target.value
+    }
+
+    const test = () => {
+        console.log(state.email)
+        console.log(state.password)
+        console.log(state.confirmPassword)
+    }
+
+    const switchToSignupPage = () => {
+        document.getElementById("submitButton").innerHTML = "Sign up"
+        document.getElementById("welcome").innerHTML = "Glad to have you here!"
+        document.getElementById("forgotLink").style.display = "none";
+        document.getElementById("createAccountLink").style.display = "none";
+        document.getElementById("backLink").style.display = "block";
+        document.getElementById("confirmTextField").style.display = "block";
```

Comment:
Can we also add text boxes for `First Name` and `Last Name` since we're storing those in the DB?